app-modules/notifications/src/Models/Concerns/HasSubscriptions.php

```diff
@@ -11,4 +11,9 @@ public function subscriptions(): MorphMany
     {
         return $this->morphMany(Subscription::class, 'subscribable');
     }
+
+    public function getSubscriptionDisplayName(): string
+    {
+        return $this->full_name;
+    }
```

Comment:
Same comment as above. I don't think we need to introduce something here since we have this with Educatable